sci-visualization/spectromatic/spectromatic-1.0-r3.ebuild

```diff
@@ -35,7 +33,3 @@ pkg_setup() {
 	tc-export CC
 	export TOPLEVEL_HOME="${EROOT}/usr"
```

Comment:
is this used by the build system? In general, `ROOT`/`EROOT` are not allowed to seep into the build system (they are not `SYSROOT`!). I would put this in `src_configure`, and replace the `EROOT` by `EPREFIX` (and also call `default` if ./configure is required).